packages/checkout/widgets-lib/src/context/view-context/BridgeViewContextTypes.ts

```diff
@@ -59,3 +66,20 @@ interface BridgeClaimWithdrawal extends ViewType {
   type: BridgeWidgetViews.CLAIM_WITHDRAWAL,
   transaction: Transaction
 }
+
+interface BridgeClaimWithdrawalInProgress extends ViewType {
+  type: BridgeWidgetViews.CLAIM_WITHDRAWAL_IN_PROGRESS,
+  data: {
+    transactionResponse: TransactionResponse;
+  }
```

Comment:
We should put `transactionResponse: TransactionResponse;` at the root level of this object rather than inside `data`. Data is an any type so we lose intellisense. 

eg:
```
interface BridgeClaimWithdrawalInProgress extends ViewType {
  type: BridgeWidgetViews.CLAIM_WITHDRAWAL_IN_PROGRESS,
  transactionResponse: TransactionResponse;
```

In retrospect I dont think this `data` field on the `ViewType` interface was ever necessary - I know we have it in other places like the `SwapInProgressView` but think this is a mistake